jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/classLoading/ClassesLoadedHandler.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics.internal.classLoading;
+
+import static io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler.defaultMeter;
+
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
+import java.time.Duration;
+import java.util.Optional;
+import jdk.jfr.consumer.RecordedEvent;
+
+public final class ClassesLoadedHandler implements RecordedEventHandler {
+  private static final String METRIC_NAME = "process.runtime.jvm.cpu.loaded_class_count";
+  private static final String EVENT_NAME = "jdk.ClassLoadingStatistics";
+  private static final String METRIC_DESCRIPTION = "Number of loaded classes";
+
+  private volatile long value = 0;
+
+  public ClassesLoadedHandler() {
+    initializeMeter(defaultMeter());
+  }
+
+  @Override
+  public void accept(RecordedEvent ev) {
+    value = ev.getLong("loadedClassCount");
+  }
+
+  @Override
+  public String getEventName() {
+    return EVENT_NAME;
+  }
+
+  @Override
+  public void initializeMeter(Meter meter) {
+    meter
+        .gaugeBuilder(METRIC_NAME)
```

Comment:
This event has `unloadedClassCount` and `loadedClassCount`, both of which seem like they would be better modeled by monotonically increasing counters instead of a gauge. While unloads may be uncommon, they do happen, and if one of the goals of this class is to give an idea of the total classes _currently_ loaded, both values should be tracked and the delta reported in the callback.

Reply:
Also, please take a look at the [OpenTelemetry JVM Metrics specification](https://github.com/open-telemetry/opentelemetry-specification/blob/7c51a04570cca391eba8926958e6bfbf5682a955/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics), which currently lists these metrics, which you should conform to:

* `process.runtime.jvm.classes.loaded`
* `process.runtime.jvm.classes.unloaded`
* `process.runtime.jvm.classes.current_loaded`

I think this handler can probably provide all 3, but if you wanted to wait an do `current_loaded` in another implementation/PR that would be fine too.

Reply:
Thank you for the review! I've changed it to use monotonically increasing counters and also added an implementation for `process.runtime.jvm.classes.current_loaded` with an `upDownCounter`. I've corrected the instrument, unit, and descriptions so that they now should conform to the specification.